molecule/default/molecule.yml

```diff
@@ -4,6 +4,47 @@ dependency:
 driver:
   name: docker
 platforms:
+  - name: ubuntu-14-04
```

Comment:
Do we want to test against 14 and 16 given they are out of LTS support?

Reply:
I don't think we should test any Ubuntu version older than 18.04.

Reply:
No strong preference. I can remove them if you want.

Reply:
I removed Ubuntu 14.04 and 16.04 and added Debina 10 and 11.

Reply:
That's perfect!  Debian 10 (oldstable) and Debian 11 (stable) are the right additions.